orchagent/portsorch.cpp

```diff
@@ -2900,6 +2974,8 @@ void PortsOrch::doPortTask(Consumer &consumer)
                         SWSS_LOG_NOTICE("Set port %s AutoNeg from %d to %d", alias.c_str(), p.m_autoneg, an);
                         p.m_autoneg = an;
                         m_portList[alias] = p;
+                        m_portStateTable.hdel(p.m_alias, "rmt_adv_speeds");
+                        updatePortStatePoll(p, PORT_STATE_POLL_AN, (an > 0));
```

Comment:
Should there be an admin status check here?

Reply:
No, the AutoNeg could be enabled prior to having the port admin up, and the port could also be admin up/down during the 5 seconds polling interval, hence it's not necessary to check the admin status here.
In the poller thread, there is always a check for the admin status.